blogs/secondary_interface/secondary_interface.md

```diff
@@ -0,0 +1,181 @@
+spec:
+  resourceName: netdeviceresource
+  nodeSelector:
+    feature.node.kubernetes.io/network-sriov.capable: "true"
+  priority: 99
+  numVfs: 50
+  nicSelector:
+    pfNames: ["ens7f0np0#25-49"]
+  deviceType: netdevice
+```
+
+5. Create the `SriovNetwork` object. This will create net-attach-def in
+  the `openshift-sriov-network-operator` namespace.
+
+```yaml
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetwork
+metadata:
+  name: sriov-test
+  namespace: openshift-sriov-network-operator
+spec:
+  resourceName: netdeviceresource
+  networkNamespace: test
+  ipam: '{ "type": "static", "addresses": [{"address": "192.168.122.71/24"}]}'
+```
+
+6. Create a test pod using the `SRIOVNetwork` object created above and
+   denoted by annotation, `k8s.v1.cni.cncf.io/networks: "sriov-test"`
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: httpd-2
+  namespace: openshift-sriov-network-operator
+  labels:
+    app: sriov
+  annotations:
+    k8s.v1.cni.cncf.io/networks: "sriov-test"
+spec:
+  containers:
+  - name: httpd
+    command: ["sleep", "30d"]
+    image: registry.redhat.io/rhel8/support-tools
+    ports:
+    - containerPort: 8080
+    securityContext:
+      allowPrivilegeEscalation: false
+      seccompProfile:
+        type: RuntimeDefault
+      capabilities:
+        drop:
+        - ALL
+```
+
+## Configuring the network observability operator to work with SR-IOV
+
+1. Deploy the network observability operator.
+
+2. Create the `FollowCollector` object with `privileged` set to `true`.
+
+```yaml
+apiVersion: flows.netobserv.io/v1beta1
+kind: FlowCollector
+metadata:
+  name: cluster
+spec:
+  agent:
+    type: EBPF
+    ebpf:
+      privileged: true
+```
+
+Network observability operator will deploy it's components (eBPF agent, flowlogs pipeline and
+console plugin), the eBPF agent will start discovering all the interfaces and attach the eBPF
+hooks, flows start being collected
+
+## Sample Network observability raw flow output by filtering on Pod VF interface `net1`
+
+By opening the console plugin and looking in the Traffic Flows table and filter by
+`Network interface name == net` will get a raw flow like the following for ICMP ping
+packets as an example
+
+```json
+{
+  "AgentIP": "10.73.116.62",
+  "Bytes": 490,
+  "DstAddr": "192.168.122.71",
+  "DstMac": "AE:39:F4:7E:03:FE",
+  "Duplicate": false,
+  "Etype": 2048,
+  "FlowDirection": "0",
+  "IcmpCode": 0,
+  "IcmpType": 0,
+  "IfDirection": 0,
+  "Interface": "net1",
+  "K8S_ClusterName": "5846f979-9d5b-4a1b-b08d-aee52e6b58a2",
+  "Packets": 5,
+  "Proto": 1,
+  "SrcAddr": "192.168.122.72",
+  "SrcMac": "06:D8:93:DB:56:9A",
+  "TimeFlowEndMs": 1694117793468,
+  "TimeFlowStartMs": 1694117789372,
+  "TimeReceived": 1694117793,
+  "app": "netobserv-flowcollector"
+}
+```
+
+<p align="center">
+  <img src="images/sriov_console.png" alt="logo" width="75%"/>
+</p>
+
+Todo: working on flow enrichment to replace those `n/a` with the proper kubernetes object string
```

Comment:
yeah .. any idea why this isn't enriched? Those IPs are not recognized in FLP ; ie. they are not referenced in the Pod specs/status ?
If so, perhaps at the moment you could take another screenshot with different columns, because it doesn't look very nice here; ie. set columns to Timestamps, source/dest IPs, Interface, bytes/pkt counters .. and whatever else you find useful

Reply:
This was icmp do no ports also we don’t have the setup anymore I was expecting pod names but I am not sure we have an rfe created to follow up https://issues.redhat.com/browse/NETOBSERV-1289

Reply:
I think for now I will remove the picture and the todo note till we have a better test and img

Reply:
added new pictures for TCP with populated fields